### ISSUE TITLE
times and description

### ISSUE THREAD

**Issue by trauber** [b1rger/carl#124]
It would be great if agenda shows start and end times and description. I can barely shell script much less code in rust. Otherwise I'd try and get together a pull request.

**Comment by b1rger** [comment]
Yes, I've thought about this too. The thing is, at least the description can be very long which might be annoying for some users. So I don't want to add it by default. And if I make it configurable, I might as well just use some kind of templating language for the agenda output, so users can configure the output how they like it. I'm not sure yet what a good solution might be

**Comment by trauber** [comment]
Maybe a command line option for including the times and description ? With a default template to go with option?
